tldextract/tldextract.py

```diff
@@ -120,6 +120,20 @@ def registered_domain(self):
             return self.domain + '.' + self.suffix
         return ''
 
+    @property
+    def fqdn(self):
+        """
+        Returns a Fully Qualified Domain Name, if there is a proper domain/suffix.
+
+        >>> extract('http://forums.bbc.co.uk/path/to/file').fqdn
+        'forums.bbc.co.uk'
+        >>> extract('http://localhost:8080').fqdn
+        ''
+        """
+        if self.domain and self.suffix:
+            return '.'.join([i for i in [self.subdomain, self.domain, self.suffix] if i])
```

Comment:
still working on tests.

i will gladly add this change, but i thought being explicit would be better for reading/maintenance.